### ISSUE TITLE
Running rebuild writes a space char to custom nginx.conf

### ISSUE THREAD

**Issue by jklmnop** [lando/lando#2383]
Using v.3.0.6 on MacOS Mojave v10.14.6 (18G5033)

```
# .lando.yml

name: xxx
recipe: drupal8
config:
  php: '7.3'
  webroot: web
  drupal: true
  xdebug: false
  database: mariadb:10.1
  via: nginx
  config:
    server: config/nginx.conf
  conf:
    php: config/php-lando.ini

env_file:
  - .env

services:
  patterns:
    type: php:7.3
    overrides:
      image: xxx.xxx.xxx.xxx/www/php_node:7.3-1.0
      ports:
        - 6006:6006

  database:
    portforward: 3308
    config: config
    creds:
      user: drupal
      password: drupal
      database: drupal_eight
  search:
    type: solr:7
    portforward: 8983
    core: lebow8
    config:
      dir: config/solr/7.x

  cache:
    type: redis:5
    portforward: 6379
    perist: true

tooling:
  redis-cli:
    service: cache
  npm:
    service: patterns
    cmd: npm
  plab-install:
    service: patterns
    cmd: npm install --prefix=/app/web/themes/custom/xxx
  plab:
    service: patterns
    cmd: npm start --prefix=/app/web/themes/custom/xxx

proxy:
  patterns:
    - patterns.xxx.lndo.site:6006
```

```
# nginx.conf

# Based on https://www.nginx.com/resources/wiki/start/topics/examples/full/#nginx-conf
user                    root root;

worker_processes  auto;
error_log         "/opt/bitnami/nginx/logs/error.log";
pid               "/opt/bitnami/nginx/tmp/nginx.pid";

events {
    worker_connections  1024;
}

http {
    include       mime.types;

    default_type  application/octet-stream;

    fastcgi_buffers 16 16k;
    fastcgi_buffer_size 32k;

    add_header    X-Frame-Options SAMEORIGIN;
    client_body_temp_path  "/opt/bitnami/nginx/tmp/client_body" 1 2;
    proxy_temp_path        "/opt/bitnami/nginx/tmp/proxy" 1 2;
    fastcgi_temp_path      "/opt/bitnami/nginx/tmp/fastcgi" 1 2;
    scgi_temp_path         "/opt/bitnami/nginx/tmp/scgi" 1 2;
    uwsgi_temp_path        "/opt/bitnami/nginx/tmp/uwsgi" 1 2;

    log_format    main '$remote_addr - $remote_user [$time_local] '
                       '"$request" $status  $body_bytes_sent "$http_referer" '
                       '"$http_user_agent" "$http_x_forwarded_for"';

    access_log    "/opt/bitnami/nginx/logs/access.log";

    sendfile           on;

    tcp_nopush         on;
    tcp_nodelay        off;

    keepalive_timeout  65;
    gzip               on;
    gzip_http_version  1.0;
    gzip_comp_level    2;
    gzip_proxied       any;
    gzip_types         text/plain text/css application/x-javascript text/xml application/xml application/xml+rss text/javascript;

    ssl_protocols      TLSv1 TLSv1.1 TLSv1.2;

    fastcgi_read_timeout 120;

    map $http_x_forwarded_proto $lando_https {
      default '';
      https on;
    }

    map $http_x_forwarded_proto $http_user_agent_https {
      default '';
      https ON;
    }

    client_max_body_size 80M;
    server_tokens off;
    include  "/opt/bitnami/nginx/conf/vhosts/*.conf";

    # HTTP Server
    server {
        # port to listen on. Can also be set to an IP:PORT
        listen  80;

        location /status {
            stub_status on;
            access_log   off;
            allow 127.0.0.1;
            deny all;
        }
    }
}
```

```bash
$ lando rebuild -y
```

Running `lando rebuild` will consistently write a space character before `user root root` in the `nginx.conf` file. This compounds after each rebuild.

```
user     root root;
        ^ right here
```

**Comment by pirog** [comment]
@jklmnop i can definitely replicate this. thanks for the report!

**Comment by pirog** [comment]
alright, this should be fixed in `3.0.7`
